secure/storage/src/value.rs

```diff
@@ -59,7 +59,7 @@ mod tests {
     #[test]
     fn ed25519_private_key() {
         let mut rng = StdRng::from_seed([13u8; 32]);
-        let value = Ed25519PrivateKey::generate_for_testing(&mut rng);
+        let value = Ed25519PrivateKey::generate(&mut rng);
```

Comment:
In theory we could use `generate_for_testing`, no need for another custom test rng